packages/facility-server/app/routes/apiv1/encounter.js

```diff
@@ -535,7 +535,26 @@ encounterRelations.get(
       model: Invoice,
     });
 
-    res.send(invoiceRecord);
+    // Convert to plain object to avoid circular references
+    const invoice = invoiceRecord.get({ plain: true });
+
+    const invoiceItemsResponse = invoice.items.map(item => {
+      const itemInsurancePlansById = keyBy(item.product?.invoiceInsurancePlanItems, 'invoiceInsurancePlanId');
+
+      const insurancePlanItems =
+        invoice?.insurancePlans?.map(({ id, code, name, defaultCoverage }) => {
+          const invoiceItem = itemInsurancePlansById[id];
+          const coverageValue = invoiceItem?.coverageValue ?? defaultCoverage;
```

Comment:
the strategy is to lookup and return the correct data from the api but do the coverage calculations on the front end to stay consistent with existing patterns